rs/src/parse/relations/mod.rs

```diff
@@ -47,7 +47,27 @@ fn parse_rel_type(x: &substrait::rel::RelType, y: &mut context::Context) -> diag
         substrait::rel::RelType::ExtensionMulti(x) => extension::parse_extension_multi_rel(x, y),
         substrait::rel::RelType::ExtensionLeaf(x) => extension::parse_extension_leaf_rel(x, y),
         substrait::rel::RelType::Cross(x) => cross::parse_cross_rel(x, y),
-        // _ => Ok(()),
+
+        // TODO: New relational nodes
+        substrait::rel::RelType::Reference(_)
+        | substrait::rel::RelType::Write(_)
+        | substrait::rel::RelType::Ddl(_)
+        | substrait::rel::RelType::HashJoin(_)
+        | substrait::rel::RelType::MergeJoin(_)
+        | substrait::rel::RelType::NestedLoopJoin(_)
+        | substrait::rel::RelType::Window(_)
+        | substrait::rel::RelType::Exchange(_)
+        | substrait::rel::RelType::Expand(_) => {
+            diagnostic!(
+                y,
+                Warning,
+                NotYetImplemented,
+                "Unimplemented relation type {}",
+                x.proto_oneof_variant()
+            );
+            Ok(())
+        } //
+          // _ => Ok(()),
```

Comment:
```suggestion
        }
```

Reply:
This is not a new comment, but one moved around before; it relates to this:
https://github.com/substrait-io/substrait-validator/blob/e325a89f0bfc294b29dcd010c17f3bdef0cba727/rs/src/parse/relations/mod.rs#L33-L36

I'd prefer it on its own line, but `rustfmt` doesn't like that.

I can clarify a bit at this point in the code? Or alternatively, I guess I could just drop it - the comment above is clear enough?

Reply:
Ah missed that. Let's just keep it like this.